src/js/items/Stack.js

```diff
@@ -433,13 +433,18 @@ lm.utils.copy( lm.items.Stack.prototype, {
 		}
 
 		for( i = 0; i < tabsLength; i++ ) {
+			if (!this.header.tabs[ i ].element.is(":visible")) { 
```

Comment:
You could've put this on the next line, and then it could've just been `if (!tabElement.is(":visible"))` which would've been much cleaner

Reply:
tabElement is used later on for placement, and needs to be set to the last visible element, so the check has to be before tabElement is set.